pkg/apis/operator/v1alpha1/const.go

```diff
@@ -62,3 +65,24 @@ var (
 		PipelineTemplatesParam: defaultParamValue,
 	}
 )
+
+var (
+	// DefaultSA is the default service account
+	DefaultSA             = "pipeline"
+	PipelineResourceName  = "pipeline"
+	TriggerResourceName   = "trigger"
+	DashboardResourceName = "dashboard"
+	AddonResourceName     = "addon"
+	ConfigResourceName    = "config"
+	ResultResourceName    = "result"
+	Interval              = 10 * time.Second
+	Timeout               = 1 * time.Minute
+)
+
+const (
+	PipelineNotReady       = "tekton-pipelines not ready"
```

Comment:
can we only move the const names ? 
those are used by webhook and controller both so we can keep at one place

`Interval`, `Timeout` and rest of them can be kept where it was before